udata/commands/purge.py

```diff
@@ -12,8 +14,11 @@
 log = logging.getLogger(__name__)
 
 
-@manager.command
-def purge(datasets=False, reuses=False, organizations=False):
+@cli.command()
+@click.option('--datasets', is_flag=True)
```

Comment:
You're usually setting the shortcut value explicitly, maybe it's a good idea to do it here and below too (`-d`...).